src/events/command-handler/CoreMessageParser.ts

```diff
@@ -9,9 +9,11 @@ export class CoreEvent extends Event<Events.Message> {
 	}
 
 	public async run(message: Message) {
-		const mentionPrefix = this.getMentionPrefix(message.content);
+		// Stop bots from running commands.
+		if (message.author.bot) return;
```

Comment:
Does this account for webhooks? I don't remember if Discord sets bot to true for them

Reply:
Yes:

```typescript
User {
  id: '737764446990237726',
  bot: true,
  username: 'GitHub',
  discriminator: '0000',
  avatar: 'df91181b3f1cf0ef1592fbe18e0962d7',
  lastMessageID: '747588398423867393',
  lastMessageChannelID: '737143305347006564'
}
```